streaming-binlog/src/main/java/org/openmrs/analytics/DebeziumListener.java

```diff
@@ -65,17 +67,8 @@ FhirConverter createFhirConverter(CamelContext camelContext) throws IOException,
 		return new FhirConverter(openmrsUtil, fhirStoreUtil, parquetUtil, params.fhirDebeziumEventConfigPath);
 	}
 	
-	private String getDebeziumConfig() {
-		return "debezium-mysql:" + params.databaseHostName + "?" + "databaseHostname=" + params.databaseHostName
-		        + "&databaseServerId=" + params.databaseServerId + "&databasePort=" + params.databasePort.intValue()
-		        + "&databaseUser=" + params.databaseUser + "&databasePassword=" + params.databasePassword
-				//+ "&name={{database.dbname}}"
-		        + "&databaseServerName=" + params.databaseName + "&databaseWhitelist=" + params.databaseSchema
-		        + "&offsetStorage=org.apache.kafka.connect.storage.FileOffsetBackingStore" + "&offsetStorageFileName="
-		        + params.databaseOffsetStorage + "&databaseHistoryFileFilename=" + params.databaseHistory + "&snapshotMode="
-		        + params.snapshotMode
-		//+ "&tableWhitelist={{database.schema}}.encounter,{{database.schema}}.obs"
-		;
+	private String getDebeziumConfigPath() {
+		return "../utils/general_config.json";
```

Comment:
@gitcliff , do you think this can construct a debezium route/path from the settings in the json fie ??

Reply:
@mozzy11 yes it should do that because i have returned a reference to the  path where the debezium json configs are defined.

Though am going to test this out give the feedback

Reply:
but there are more settings in the json file more than just the Dbezium path configs

Reply:
yes @mozzy11  the  intention is to have one json schema with all the necessary pipeline properties and configs as requested here https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/issues/51

Reply:
How is this supposed to work?

Reply:
@gitcliff  , ideally this cant work , you must implement some logic of how the debeziium route is to be constructed from the settings in the json file